README.md

```diff
@@ -1,53 +1,66 @@
 # mkdocs-redirects
-Open source plugin for Mkdocs page redirects
+
+Plugin for [`mkdocs`](https://www.mkdocs.org/) to create page redirects (e.g. for moved/renamed pages).
 
 ## Installing
 
-> **Note:** This package requires MkDocs version 1.0.4 or higher. 
+> **Note:** This package requires MkDocs version 1.0.4 or higher.
 
 Install with pip:
 
 ```bash
 pip install mkdocs-redirects
 ```
 
-Enable the plugin in your `mkdocs.yml`:
+## Using
+
+To use this plugin, specify your desired redirects in the plugin's `redirect_maps` setting in your `mkdocs.yml`:
 
 ```yaml
 plugins:
-    - search
```

Comment:
You need the `search` plugin or it won't work!  There's some implicit dependency (@brian-rickman might remember what it was).

Reply:
I made a note underneath this code block pointing to the MkDocs doc that explains just that.

Reply:
Oh, you mean that this plugin itself requires the `search` plugin? It shouldn't...

Reply:
It does though: https://github.com/datarobot/mkdocs-redirects/pull/1

Let me try to recall why...

Reply:
![image](https://user-images.githubusercontent.com/177557/63125944-d1e7f380-bf7c-11e9-853e-b675b61ba165.png)

Reply:
So its not required, but will lead to very confusing behavior if you just copy/paste the example without `search`.  RTFM, of course, but let's be honest who does that? ;)

Reply:
Yeah, I know the section. It's just saying that if you specify the `plugins:` section in your config, the default will be overridden. There's nothing in this plugin that _requires_ the `search` plugin. If someone wants to run a site without `search`, they don't need to include it.

It's just that if this is the first/only plugin someone uses, they'll have to explicitly include the defaults (I believe just `search`) along with this plugin.

I included a note below the code block reminding users of that fact.

Reply:
> very confusing behavior if you just copy/paste 

That should teach the user to not blindly copy-paste anything off the internet 😝

Reply:
Fair enough, I just remember we lost time trying to figure it out when this mysteriously happened to us.